src/pages/gen2/start/index.mdx

```diff
@@ -0,0 +1,237 @@
+
+```text
+├── amplify/
+│   ├── auth/
+│   │   └── resource.ts
+│   ├── data/
+│   │   └── resource.ts
+│   ├── backend.ts
+│   └── package.json
+├── node_modules/
+├── .gitignore
+├── package-lock.json
+├── package.json
+└── tsconfig.json
+```
+
+Once scaffolded, you're ready to create your first cloud sandbox. Amplify gives every developer a personal cloud sandbox where they can test changes rapidly using live AWS services. Sandbox should not be used for production applications. To use Amplify's [Sandbox mode](http://localhost:3000#fix-me), run the following command:
+
+<NodePackageManagerTabs
+  npm="npx amplify sandbox"
+  yarn="yarn amplify sandbox"
+  pnpm="pnpm amplify sandbox"
+/>
+
+Try making a change to your data model and hit save. Changes are deployed in seconds.
+
+```ts {6} showLineNumbers
+import { type ClientSchema, a, defineData } from '@aws-amplify/backend';
+
+const schema = a.schema({
+  Todo: a.model({
+    title: a.string(),
+    description: a.string(),
+    priority: a.enum(['low', 'medium', 'high']).default('low')
+  })
+});
+
+export type Schema = ClientSchema<typeof schema>;
+
+export const data = defineData({
+  schema
+});
+```
+
+## Authentication
+
+Each of the `resource.ts` files is where you _define_ the corresponding backend resource:
+
+```ts title="amplify/auth/resource.ts"
+import { defineAuth } from '@aws-amplify/backend';
+
+export const auth = defineAuth({
+  loginWith: {
+    email: true
+  }
+});
+```
+
+The definitions are then imported and set in the `backend`:
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from '@aws-amplify/backend';
+import { auth } from './auth/resource.js';
+import { data } from './data/resource.js';
+
+defineBackend({
+  auth,
+  data
+});
+```
+
+### Add auth to your frontend
+
+First, install the Amplify UI component library:
+
+<NodePackageManagerTabs
+  npm="npm add @aws-amplify/ui-react"
+  yarn="yarn add @aws-amplify/ui-react"
+  pnpm="pnpm add @aws-amplify/ui-react"
+/>
+
+Next, in your app's entry point (e.g. `pages/_app.ts`, etc.), add the following imports and wrap your App in the `withAuthenticator` function:
+
+```ts title="pages/_app.ts"
+import { withAuthenticator } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import config from '@/amplifyconfiguration.json';
+import '@aws-amplify/ui-react/styles.css';
+import '@/styles/globals.css';
+import type { AppProps } from 'next/app';
+
+// configure the Amplify client library with the configuration generated by `amplify sandbox`
+Amplify.configure(config);
+
+function App({ Component, pageProps }: AppProps) {
+  return <Component {...pageProps} />;
+}
+
+export default withAuthenticator(App);
+```
+
+Run your application with `npm run dev` and navigate to `http://localhost:3000`. You should now see the authenticator, which is already configured and ready for your first sign-up!
+
+## Data
+
```

Comment:
Should add small text markers to transition the walk through / steps. Can be simple but for example, "Next you will want to set up your data resources to to save and read data to and from the cloud."

Reply:
added more of an intro here